substrate/frame/staking/src/pallet/impls.rs

```diff
@@ -575,7 +575,7 @@ impl<T: Config> Pallet<T> {
 
 			let era_duration = (now_as_millis_u64.defensive_saturating_sub(active_era_start))
 				.saturated_into::<u64>();
-			let staked = Self::eras_total_stake(&active_era.index);
+			let staked = Self::eras_total_stake(active_era.index);
```

Comment:
```suggestion
			let staked = ErasTotalStake::<T>::get(&active_era.index);
```
Part of the aim here is to replace any usage of `Self::storage()` with `Storage::<T>::get()` in the sdk codebase

Reply:
Now it's done. I replaced these usages everywhere I was able to spot them.